src/Raygun.Blazor.Maui/Raygun.Blazor.Maui.csproj

```diff
@@ -1,9 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>net8.0-android;net8.0-ios;net8.0-maccatalyst</TargetFrameworks>
+        <TargetFrameworks>net8.0-android</TargetFrameworks>
         <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">
-            $(TargetFrameworks);net8.0-windows10.0.19041.0</TargetFrameworks>
+            $(TargetFrameworks);net8.0-ios;net8.0-maccatalyst;net8.0-windows10.0.19041.0</TargetFrameworks>
+        <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('osx'))">
+            $(TargetFrameworks);net8.0-ios;net8.0-maccatalyst</TargetFrameworks>
```

Comment:
When the platform is Windows, it builds for Windows, Android, Mac and iOS. When the platform is osx (macos) it builds for Android, iOS and Mac. On the rest of platforms (basically Linux) it only builds for Android.

Same for the Sample project.